crates/router/src/connector/globalpay/requests.rs

```diff
@@ -1,13 +1,19 @@
+use common_utils::types::StringMinorUnit;
 use masking::Secret;
 use serde::{Deserialize, Serialize};
 
+#[derive(Debug, Serialize)]
+pub struct GlobalPayRouterData<T> {
+    pub amount: StringMinorUnit,
+    pub router_data: T,
+}
 #[derive(Debug, Serialize)]
 pub struct GlobalpayPaymentsRequest {
     /// A meaningful label for the merchant account set by Global Payments.
     pub account_name: Secret<String>,
     /// The amount to transfer between Payer and Merchant for a SALE or a REFUND. It is always
     /// represented in the lowest denomiation of the related currency.
-    pub amount: Option<String>,
+    pub amount: Option<StringMinorUnit>,
```

Comment:
Are we you sure, globalPay works in minorUnit and not major, if not can you verify this with someone!

Reply:
yeah it works in the lowest denomiation of the related currency